dashboards/active_job/main.json

```diff
@@ -4,12 +4,12 @@
   ],
   "dashboard": {
     "title": "Active Job",
-    "description": "This dashboard gives an overview of Active Job jobs. Checkout queue lengths and the status per job.",
+    "description": "This dashboard gives an overview of Active Job jobs. View queue lengths and the status per queue and priority.",
     "visuals": [
       {
-        "title": "Overall job status",
-        "description": "Job status reported by Active Job",
-        "line_label": "%name% %queue%-%status%",
+        "title": "Job status per queue",
+        "description": "Processed and failed jobs per queue.",
+        "line_label": "%queue% queue - %status%",
```

Comment:
I decided not to show the metric name here as it has grown quite long and doesn't contribute much. Now we can make the line label a bit more human friendly.